x-pack/legacy/plugins/spaces/public/views/management/edit_space/enabled_features/feature_table.tsx

```diff
@@ -102,9 +102,8 @@ export class FeatureTable extends Component<Props, {}> {
             id={record.feature.id}
             checked={checked}
             onChange={this.onChange(record.feature.id) as any}
-            aria-label={
-              checked ? `${record.feature.name} visible` : `${record.feature.name} disabled`
-            }
+            label={checked ? `${record.feature.name} visible` : `${record.feature.name} disabled`}
```

Comment:
Can remove this conditional and leave it at `${record.feature.name} visible`. The checked state will be communicated by assistive tech letting users know what's going on.

As a separate side note, using "disabled" in the label can be confusing because the control isn't actually "disabled".